docs/rfc/rfc-1.adoc

```diff
@@ -184,22 +184,33 @@ reason), all they need to do is not submit their reveal and wait 30 days.
 
 [[sweeping]]
 ==== Sweeping
+
+Governable Parameters
+
+- `sweeping_refund_safety_time`: The amount of time prior to when a UTXO
+  becomes eligible where we will not include it in a sweeping transaction.
+- `sweep_period`: The amount of time we wait between sweeps.
+- `sweep_max_btc`: The amount of total bitcoin that will trigger an early sweep.
+- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
+  be included in a sweep.
+
 The operators sign a transaction that unlocks all revealed deposits, combines
```

Comment:
It would be good to update this sentence at some point (not necessarily in this PR, maybe in #56?), as it is no longer true - not all revealed deposits will be consolidated into one UTXO, only those who qualify based on the declared `max_fee`.

Reply:
deal!